locales/templates/app.pot

```diff
@@ -495,6 +495,10 @@ msgid ""
 "wants to access your location."
 msgstr ""
 
+msgctxt "geolocation_dialog_message_cache_it"
+msgid "Don't ask again for this site until exit"
```

Comment:
I've been turning this string around in my head since this morning and it's been bothering me for some reason - so I asked around, and got the confirmation that it sounded kinda wrong.
A suggestion I got was:
* "Don't ask for this site again during this session"
There are probably other ways to say this too - it's just it sounds strange to say "until exit" together like that.
Note the string is pretty long already, and probably going to be longer in many of the locales we're shipping in, so hopefully it will fit in :)

Reply:
Hi Delphine, I am not sure users can understand "this session"? What I want to describe the meaning is like "before closing Rocket" or "until exit Rocket" or "until leaving Rocket"... 
If you think "this session" is better so far and no other better candidates, I agree with using it instead of using "until exit"

Thanks for your help and struggle :)

Reply:
Another concern is how a session is defined may be very different (From an engineering perspective, at least.). If a user merely goes to background and return later, the permission might still be held if the App is not killed by the system. In this case, some users may have expected going background to be an end of a session. Yet another example is around tabs: A user might close a tab and expected that to be the end of the session while per current implementation it is not.

Reply:
another reason we use the term "exit" is because there is an "exit" button on Rocket

Reply:
`Don’t ask again for this site until {Rocket} restarts/is restarted`? Firefox has a [similar string](https://transvision.mozfr.org/?sourcelocale=en-US&locale=zh-TW&repo=gecko_strings&search_type=entities&recherche=browser/browser/preferences/preferences.ftl:permissions-notification-pause.label&entire_string=entire_string).

Reply:
That sounds like a nice one! I'll bring it to the discussion.

Reply:
great suggestion. I discussed with Joe and let's go with  "Don’t ask again for this site until {Rocket} restarts"